src/EnergyPlus/HVACManager.cc

```diff
@@ -146,42 +141,13 @@ void ManageHVAC(EnergyPlusData &state)
     //  manage calls to Predictor and Corrector and other updates in ZoneTempPredictorCorrector
     //  manage variable time step and when zone air histories are updated.
 
-    // Using/Aliasing
-    using DemandManager::ManageDemand;
-    using DemandManager::UpdateDemandManagers;
-    using EMSManager::ManageEMS;
-    using InternalHeatGains::UpdateInternalGainValues;
-    using NodeInputManager::CalcMoreNodeInfo;
-    using OutAirNodeManager::SetOutAirNodes;
-    using OutputReportTabular::GatherComponentLoadsHVAC;
-    using OutputReportTabular::UpdateTabularReports; // added for writing tabular output reports
-    using PollutionModule::CalculatePollution;
-    using RefrigeratedCase::ManageRefrigeratedCaseRacks;
-    using ScheduleManager::GetCurrentScheduleValue;
-    using SizingManager::UpdateFacilitySizing;
-    using SystemAvailabilityManager::ManageHybridVentilation;
-    using SystemReports::InitEnergyReports;
-    using SystemReports::ReportSystemEnergyUse;
-    using WaterManager::ManageWater;
-    using WaterManager::ManageWaterInits;
-    using ZoneContaminantPredictorCorrector::ManageZoneContaminanUpdates;
-    using ZoneEquipmentManager::CalcAirFlowSimple;
-    using ZoneEquipmentManager::UpdateZoneSizing;
-    using ZoneTempPredictorCorrector::DetectOscillatingZoneTemp;
-    using ZoneTempPredictorCorrector::ManageZoneAirUpdates;
-
     // SUBROUTINE PARAMETER DEFINITIONS:
     static constexpr std::string_view EndOfHeaderString("End of Data Dictionary");                          // End of data dictionary marker
     static constexpr std::string_view EnvironmentStampFormatStr("{},{},{:7.2F},{:7.2F},{:7.2F},{:7.2F}\n"); // Format descriptor for environ stamp
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
     Real64 PriorTimeStep;       // magnitude of time step for previous history terms
     Real64 ZoneTempChange(0.0); // change in zone air temperature from timestep t-1 to t
-    int NodeNum;
-    bool ReportDebug;
-    int ZoneNum;
-
-    bool DummyLogical;
```

Comment:
[src/EnergyPlus/HVACManager.cc:180]:(style),[variableScope],The scope of the variable 'NodeNum' can be reduced.
[src/EnergyPlus/HVACManager.cc:181]:(style),[variableScope],The scope of the variable 'ReportDebug' can be reduced.
[src/EnergyPlus/HVACManager.cc:184]:(style),[variableScope],The scope of the variable 'DummyLogical' can be reduced.